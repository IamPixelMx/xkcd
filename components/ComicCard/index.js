import ComicCard from './ComicCard';

export { ComicCard };
