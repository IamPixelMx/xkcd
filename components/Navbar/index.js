import Navbar from './Navbar';
import NavbarInput from './NavbarInput';
import NavbarItem from './NavbarItem';

export { Navbar, NavbarInput, NavbarItem };
