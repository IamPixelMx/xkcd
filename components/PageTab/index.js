import PageTab from './PageTab';

export { PageTab };
