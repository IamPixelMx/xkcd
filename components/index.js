import { ComicCard } from './ComicCard';
import { Footer } from './Footer';
import { Navbar, NavbarInput, NavbarItem } from './Navbar';
import { PageTab } from './PageTab';
import { Pagination } from './Pagination';

export { ComicCard, Footer, Navbar, NavbarInput, NavbarItem, PageTab, Pagination };
