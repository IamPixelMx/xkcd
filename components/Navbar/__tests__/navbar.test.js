import React from 'react';
import renderer from 'react-test-renderer';
import { Navbar } from '../../PageTab/__tests__/node_modules/components';

describe('Navbar', () => {
  const navbarElement = renderer.create(<Navbar />).toJSON();

  it('Renders Navbar', () => {
    expect(navbarElement).toMatchSnapshot();
  });
});
