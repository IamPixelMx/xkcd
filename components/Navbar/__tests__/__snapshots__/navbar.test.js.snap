// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Renders Navbar 1`] = `
<nav
  className="navbar"
>
  <div
    className="container"
  >
    <div
      className="navbar-brand"
    >
      <figure
        className="image is-96x96"
      >
        <img
          alt="xkcd-logo"
          src="/xkcd.png"
        />
      </figure>
      
      <span
        className="navbar-burger burger "
        data-target="navbarMenuHeroB"
        onClick={[Function]}
      >
        <span />
        <span />
        <span />
      </span>
    </div>
    <div
      className="navbar-menu "
      id="navbarMenuHeroB"
    >
      <div
        className="navbar-end "
      >
        <a
          className="navbar-item "
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Inicio
        </a>
        <a
          className="navbar-item "
          href="/favorites"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Favoritos
        </a>
      </div>
    </div>
  </div>
</nav>
`;
